presto-main/src/main/java/com/facebook/presto/cost/ComparisonStatsCalculator.java

```diff
@@ -33,179 +33,215 @@
 {
     private ComparisonStatsCalculator() {}
 
-    public static Optional<PlanNodeStatsEstimate> comparisonExpressionToLiteralStats(
+    public static Optional<PlanNodeStatsEstimate> estimateExpressionToLiteralComparison(
             PlanNodeStatsEstimate inputStatistics,
-            Optional<Symbol> symbol,
-            SymbolStatsEstimate expressionStats,
-            OptionalDouble doubleLiteral,
-            ComparisonExpression.Operator operator)
+            SymbolStatsEstimate expressionStatistics,
```

Comment:
The symbol can be missing in case of a complex expression. Just statistics is confusing. As there is another parameter that represents node statistics.